smartsim/settings/arguments/launch/slurm.py

```diff
@@ -349,5 +351,5 @@ def set(self, key: str, value: str | None) -> None:
             )
             return
         if key in self._launch_args and key != self._launch_args[key]:
```

Comment:
Whoops! I lied!! Last one for real
```suggestion
        if key in self._launch_args and value != self._launch_args[key]:
```